### ISSUE TITLE
Disabling Launch AdGuard for Safari at Login Doesn't Work

### ISSUE THREAD

**Issue by Haravikk** [AdguardTeam/AdGuardForSafari#370]
### Prerequisites

- [ X ] I am running the latest version
- [ X ] I checked the documentation and found no answer
- [ X ] I checked to make sure that this issue has not already been filed

### Issue Details

* AdGuard for Safari version: 1.7.4
* Environment name and version: Safari 13.1.2
* Operating System: macOS 10.15.6
* Filters you use in AdGuard: Basic
* Any other browser extensions you have installed: 1Password

### Expected Behavior
Disabling the Launch AdGuard for Safari at Login option should prevent AdGuard from launching at login.

### Actual Behavior
As long as the option has been enabled at least once, AdGuard for Safari will always launch at login, even if the setting is later disabled.

### Additional Information
This one is driving me nuts; I use multiple user accounts on my system for different purposes, but didn't really need AdGuard for Safari running on one of them so tried to disable it, but it opens on login every single time anyway, even with the extensions disabled in Safari (in fact that's worse, as it then complains about it on login every. single. time.).

I can't even see **how** AdGuard is launching at login, it's not listed in login items, and I don't see a launch agent for it. How can I prevent it from opening?

**Comment by zzebrum** [comment]
@Haravikk please, try the latest beta version: https://agrd.io/safari_beta - it should be ok there.

**Comment by zzebrum** [comment]
@Haravikk @dorjanbenic hi! any updates on the issue?

**Comment by dorjanbenic** [comment]
@zzebrum Sorry to disappoint you but I did not since it was not big issue for me because I rarely restart my Mac. I reported it so you guys are informed about potential issue, it did not bother me much.

I also had many question:
1. How will it interact with app that I installed from App Store?
2. Will I need to delete then original app from the App Store?
3. How will I update beta app then once I install it?

...so I opted to move on and wait for next MAS release.

**Comment by zzebrum** [comment]
@dorjanbenic MAS release is out now and you can update the app :)

**Comment by dorjanbenic** [comment]
AdGuard is still not starting with may Mac. Safari AdGuard button is showing that AdGuard is disabled when I start it. Wording is also wrong in my opinion. It implies that AdGuard is disabled and it wont block anything while that is not true it just wont auto update filters. I missed small text first time.

I think that you might replace text **"AdGuard is disabled.  It needs to be running..."**  with another checkbox below **Enabled. on this website** that would write **Auto update filters**.  

Then you might need it running for **Block an element** picker? 

![Screen Shot 2020-08-27 at 17 13 42](https://user-images.githubusercontent.com/39588255/91461415-42606a80-e889-11ea-9b6a-270ba8cc57eb.png)

**Comment by zzebrum** [comment]
@dorjanbenic and `Launch AdGuard at Login` is enabled?

**Comment by dorjanbenic** [comment]
@zzebrum I have created screenshot of my AdGuard setting as well as AdGuard button once my Mac have been turned one and all application have been loaded.

**Comment by zzebrum** [comment]
@dorjanbenic can you check if AdGuard for Safari is present at `Login items` under system `settings - Users&groups`?

**Comment by dorjanbenic** [comment]
Before I reported issue here on version 1.7.4 I have tried to add AdGuard in 'Login items' manually and it worked as expected. I have removed that option since I needed to close AdGuard window in that case which is one click as same as making it start via AdGuard button in Safari. 

So to answer your question AdGuard is not present in 'Login items' in 1.7.4 & 1.8.2. If I add ti manually than it works but I need to close window after it started up. Hide checkbox does not seems to work.

I don't believe that there should be AdGuard in 'Login items' even if I have login option enabled because I have 6 app that I have enabled to start up automatically with my Mac and they are not present in 'Login items'. My login items list was always empty.

Apps that I have enabled with login option:
1. Micro Snitch
2. Little Snitch
3. ProtonVPN
4. ColorSnapper2
5. PasteBox
6. Amphetamine
